docs/getting-started/first-traffic-split.md

```diff
@@ -150,54 +151,82 @@ Lets split traffic between our two Revisions:
-        NAME            SERVICE   TRAFFIC   TAGS   GENERATION   AGE   CONDITIONS   READY   REASON
-        hello-knative   hello     50%              2            10m   3 OK / 4     True
-        hello-world     hello     50%              1            36m   3 OK / 4     True
-        ```
 
-=== "kubectl"
-     Though the following example doesn't cover it, you can peak under the hood to Kubernetes to see the revisions as Kubernetes sees them by running the command:
-    ```bash
-    kubectl get revisions
+
+Access your Knative service multiple times on the browser to see the different output being served by each Revision.
+
+Similarly, you can access the Service URL from the terminal multiple times to see the traffic being split between the Revisions.
+```bash
+echo "Accessing URL $(kn service describe hello -o url)"
+curl "$(kn service describe hello -o url)"
+```
+
+!!! Success "Expected output"
+    ```{ .bash .no-copy }
+    Hello Knative!
+    Hello World!
+    Hello Knative!
+    Hello World!
     ```
 
+### Tag based routing
+
+Lets say you still have http clients using the old revision `hello-00001` and they can't use the new revision `hello-00002` just yet as they might be in a middle of a user session.
 
-Access your Knative service on the browser again [http://hello.default.127.0.0.1.sslip.io](http://hello.default.127.0.0.1.sslip.io){target=_blank}, and refresh multiple times to see the different output being served by each Revision.
+You can tag a revision and this will create a new url exclusively for the revision.
 
-Similarly, you can `curl` the Service URL multiple times to see the traffic being split between the Revisions.
+Lets tag the old revision `hello-00001` as `v1` using the `kn` CLI
 ```bash
-curl http://hello.default.127.0.0.1.sslip.io
+kn service update hello --tag hello-00001=v1
+```
+
+You can see the revision now was a tag under the column `TAGS`
+```bash
+kn revision list hello-00001
 ```
 
 !!! Success "Expected output"
     ```{ .bash .no-copy }
-    curl http://hello.default.127.0.0.1.sslip.io
-    Hello Knative!
+    NAME          SERVICE   TRAFFIC   TAGS   GENERATION   AGE   CONDITIONS   READY   REASON
+    hello-00001   hello     50%       v1     1            15m   3 OK / 4     True
+    ```
 
-    curl http://hello.default.127.0.0.1.sslip.io
+You can append `v1-` to the Knative Service domain (eg. `http://v1-hello.default.${LOADBALANCER_IP}.sslip.io`) to access this revision, you can get the value of url for the tagged revision from the `status` field on the Kubernetes resource `ksvc`. You can use kubectl to print the yaml representation to read `status`. Use the following `curl` to access the revision
+```bash
+echo "Accessing URL $(kubectl get ksvc hello  -o yaml | grep '//v1-' | awk -F 'url: ' '{print $2}')"
+curl "$(kubectl get ksvc hello  -o yaml | grep '//v1-' | awk -F 'url: ' '{print $2}')"
+```
+
+!!! Success "Expected output"
+    ```{ .bash .no-copy }
     Hello World!
     ```
```

Comment:
I think showing `Accessing URL http://v1-hello.default.${LOADBALANCER_IP}.sslip.io` this on the doc is confusing, but they will see the real value when they run it, this is just in case they have problems they can see the actual URL being called.

Reply:
My thinking was that when I pasted the commands into the terminal I didn't see `Hello World!` immediately I just saw `Accessing URL...`. I then remembered I had copied 2 commands and had still had to run the second one. 

This might be because I've run the tutorial before so I was used to hello world outputting immediately.

Reply:
I want to avoid the noise in the doc with so much repetitive lines with `${LOADBALANCER_IP}`